docker/scripts/setup_env

```diff
@@ -33,11 +56,33 @@ else
     START_MYSQL=${START_MYSQL:-true}
 fi
 
+# Initialize values that might be stored in a file
+file_env 'DATABASE_PASSWORD' ${HUGINN_DATABASE_PASSWORD:-${DATABASE_PASSWORD:-password}}
```

Comment:
This passes the default value taken from the original variable definition from Line 85 (now commented out)

Reply:
This might need to be defined for every var depending on the behavior when passing in values via other means (env vars, .evn-file, etc)

Reply:
> This might need to be defined for every var depending on the behavior when passing in values via other means (env vars, .evn-file, etc)

this appears to not be a problem, though I can't say I fully understand _why_ it's not a problem.   But in my testing I changed variable not passed to the `file_env` function just fine via the different methods.